parser/Parser/CAFF_HEADER.h

```diff
@@ -1,10 +1,16 @@
 #pragma once
 #include "Block.h"
-class CAFF_HEADER :
-    public Block
+#include <cstdint>
+
+class CAFF_HEADER : public Block
 {
-    char magic[4];
-    unsigned int headerSize;
-    unsigned int animationNumber;
+public:
+
+    char m;
+    char a;
+    char g;
+    char ic;
```

Comment:
The original works as well.
```suggestion
    char magic[4];
```

Reply:
Reverted in commit [bdbb580](https://github.com/radl97/BiztonsagosGang-2021/pull/1/commits/bdbb58027f9c6725c4c7c9de7d0278999633fdcc).